schema.sql

```diff
@@ -16,4 +16,7 @@ CREATE TABLE IF NOT EXISTS public.animals
 )
 
 ALTER TABLE IF EXISTS public.animals
-    OWNER to postgres;
\ No newline at end of file
+    OWNER to postgres;
+
+alter table animals
+add species character(20);
```

Comment:
- [ ] Kindly note that I am not able to create your table because on your ID generation it is throwing out this error below :point_down: 

![image](https://user-images.githubusercontent.com/65068771/155528648-8439419b-ad39-412a-8c45-66d39fe06e84.png)

- [ ] I solve this by generating the animal ID in this way :point_down: 

![image](https://user-images.githubusercontent.com/65068771/155528811-ea2b2133-03e3-4856-b048-40e6559189e5.png)